source/appModules/totalcmd.py

```diff
@@ -38,9 +38,11 @@ def event_gainFocus(self):
 			while obj and obj.previous and obj.windowClassName!="Window":
 				obj=obj.previous
 			try:
-				if obj2.parent.parent.previous.firstChild.role  == 14:
+				if obj2.parent.parent.previous.firstChild.role  == controlTypes.ROLE_LIST:
+					# Translators: the word left for the left window in your language (only the word left).
```

Comment:
To clarify, do you mean "a message telling the user that they reached the left most column in this window" or something similar?